videodev2.h is needed in userspace by Video4Linux programs. 
linux/time.h has a definition of 'struct timeval' which conflicts with the
one in time.h, even though it has the same fields with types of the same
lengths.  So just use the userspace one in userspace!

Update 2005-05-05 gotom.

--- include.orig/linux/videodev2.h	2005-04-21 09:03:16.000000000 +0900
+++ include/linux/videodev2.h	2005-05-05 18:21:52.639776621 +0900
@@ -15,6 +15,8 @@
  */
 #ifdef __KERNEL__
 #include <linux/time.h> /* need struct timeval */
+#else
+#include <time.h> /* need struct timeval */
 #endif
 #include <linux/compiler.h> /* need __user */
 
